### PR TITLE
Complete Command Test + Storage Tests

### PR DESCRIPTION
**Pull request by akd13**
Added tests for Complete Command and StorageManager (as well as methods missed in coveralls)

### CODE REVIEW

**Review by ChuaPingChan — APPROVED** [review]

File: src/main/java/seedu/multitasky/model/StorageUserPrefs.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    public String getEntryBookFilePath();
+
+    public void setEntryBookFilePath(String entryBookFilePath);
+
+    /**
+     * The getter and setter for the snapshot file path
+     */
+    public static String getEntryBookSnapshotPath() {
+        return UserPrefs.getEntryBookSnapshotPath();
```
Comment (kevinLamKB) [inline src/main/java/seedu/multitasky/model/StorageUserPrefs.java]: not recommended to use UserPrefs static methods here. this forces StorageUserPrefs interface to be unnecessarily dependent on UserPrefs. i think just doing a public static methodname(); is enough for all the methods in the interface, let the classes that implement the interface override how they want the method to be implemented.

**Review by kevinLamKB — APPROVED** [review]

File: src/main/java/seedu/multitasky/model/StorageUserPrefs.java (hunk 1 above)
Comment (akd13) [reply src/main/java/seedu/multitasky/model/StorageUserPrefs.java]: Static methods cannot be overriden,I already tried it. You cannot declare static methods in an interface without defining them or changing the method signature to non-static (which is difficult to do as the Storage class needs to keep track of the snapshots for each instance)
It was either being dependent on UserPrefs or leaving the implementations blank, and Codacy was giving an error if i left the fields blank.

Comment (akd13) [reply src/main/java/seedu/multitasky/model/StorageUserPrefs.java]: https://stackoverflow.com/questions/2223386/why-doesnt-java-allow-overriding-of-static-methods

**Review by ChuaPingChan — APPROVED** [review]

**Review by mattheuslee — APPROVED** [review]

**Comment by mattheuslee** [comment]
@akd13 please resolve the conflicts and merge

**Pull request merged**